game/scripts/npc/heroes/shadow_shaman.txt

```diff
@@ -11,7 +11,7 @@
     "Ability11"   "special_bonus_movement_speed_25"
     "Ability12"   "special_bonus_cast_range_100"
     "Ability13"   "special_bonus_all_stats_8"   //OAA
-    "Ability14"   "special_bonus_armor_8"   //OAA
+    "Ability14"   "special_bonus_unique_shadow_shaman_4" // +4 Wards Summoned
     "Ability15"   "special_bonus_unique_shadow_shaman_2"
     "Ability16"   "special_bonus_unique_shadow_shaman_1" // +1 Mass Serpent Ward Attacks Health
     "Ability17"   "special_bonus_unique_shadow_shaman_3" // +3s Shackles Duration
```

Comment:
You forgot about this. Should be Shackles Duration, Ward Health, and +350 Ether Shock Damage in order. Apparently the labels weren't moved when the Talents got shuffled for a previous patch parity.